plugins/ufm_consumer_plugin/build/Dockerfile

```diff
@@ -0,0 +1,65 @@
+FROM ubuntu:20.04
```

Comment:
why no to be based on Ubuntu22? like UFM container...

Reply:
because it is based on basic "hello word" plugin with UI support